region/brinstar/kraid/Warehouse Kihunter Room.json

```diff
@@ -558,6 +558,10 @@
       "exitCondition": {
         "leaveWithSpark": {}
       },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
```

Comment:
might be possible to lay a PB by the save door, but would need to be its own strat which has access to 3

Reply:
Makes sense, I was thinking we could do another pass through all the rooms later to add strats like that